src/synthesizer/abundances/reference_abundance_patterns.py

```diff
@@ -54,6 +54,56 @@ class Asplund2009:
     }
 
 
+class GalacticConcordance:
+
+    """
+    The Galactic Concordance pattern from Nicholls et al.
+    """
```

Comment:
This is also not a properly initialised class. I'm also confused why this is independent of the abundance_patterns submodule, shouldn't this inherit from that parent class? Each of these is an independent new class

Reply:
No, these just define basic information about a reference abundance pattern, e.g. a solar pattern. These would probably be better as data classes but I don't consider it a priority to convert them.

Reply:
Then these shouldn't be classes. 

I think it is a priority. We aren't merging more "for now" code.